cluster-state-service/handler/event/kinesis_consumer_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package event
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"testing"
+	"github.com/golang/mock/gomock"
+	"github.com/blox/blox/cluster-state-service/handler/mocks"
+	"github.com/aws/aws-sdk-go/service/kinesis"
+	"context"
+)
+
+const (
+	streamName       = "test"
+	kinesisMessageBody1 = "messageBody"
+	kinesisMessageBody2 = "messageBody2"
+)
+
+type consumerMockKinesisContext struct {
+	mockCtrl             *gomock.Controller
```

Comment:
Nit - The formatting here looks weird. [gofmt](https://atom.io/packages/gofmt) should help with this.